lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -722,6 +744,17 @@ instance Arbitrary (PostTransactionFeeData t) where
     arbitrary = genericArbitrary
     shrink = genericShrink
 
+instance Arbitrary PostExternalTransactionData where
+    arbitrary = do
+        count <- choose (0, 32)
+        bytes <- BS.pack <$> replicateM count arbitrary
+        return $ PostExternalTransactionData bytes
+    shrink (PostExternalTransactionData bytes) | BS.null bytes = []
```

Comment:
I was wondering if it would be easier to just use the `instance Arbitrary [Char]` here. Like
```haskell
shrink (PostExternalTransactionData bytes) = PostExternalTransactionData . BS.pack <$> shrink (BS.unpack bytes)
```

Reply:
make sense. fixed